web-react/src/pages/directory/display/DetailsFellowship.jsx

```diff
@@ -42,10 +42,13 @@ const DetailsFellowship = () => {
     })
   )
 
-  const check = last3Weeks()?.map((number) => {
+  const check = last3Weeks()?.forEach((number) => {
```

Comment:
This is not making sense. Did you merge from deploy before pushing these updates?